pom.xml

```diff
@@ -1,9 +1,86 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.openmrs</groupId>
-  <artifactId>openmrs-content-hiv</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <name>Archetype - openmrs-content-hiv</name>
-  <url>http://maven.apache.org</url>
+	<modelVersion>4.0.0</modelVersion>
+ 	<groupId>org.openmrs.content</groupId>
```

Comment:
The formatting here is a little inconsistent using both spaces and tabs. Could we stick to one please?